managed/services/grafana/auth_server.go

```diff
@@ -108,14 +108,16 @@ var rules = map[string]role{
 	// "/" is a special case in this code
 }
 
-var vmProxyPrefixes = []string{
-	"/graph/api/datasources/proxy/1/api/v1/",
+var lbacPrefixes = []string{
+	"/graph/api/datasources/uid",
 	"/graph/api/ds/query",
-	"/graph/api/v1/labels",
+	// "/graph/api/v1/labels",
```

Comment:
Is it commented out and not removed to keep it visible for some reason?

Reply:
Correct. I checked the logs and I saw no requests being made against that URL. Therefore, to save on processing, I commented it out but would like to keep it for some time.

Reply:
Maybe it would be good to write it there. But up to you.

Reply:
Done.